domains/shapesdocs.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Syst69",
+           "email": "xcrosssaroj@gmail.com",
+           "discord": "750339984598368287"
+        },
+    
+        "record": {
+            "AAAA": "fanmade docs for creating shapes"
```

Comment:
Invalid AAAA records, should be valid array populated by strings containing valid IPv6 addresses.